frontend/src/main.ts

```diff
@@ -24,4 +24,8 @@ app.component('TabPanel', TabPanel)
 app.component('Tree', Tree)
 app.component('RadioButton', RadioButton)
 app.component('Card', Card)
+app.use(VueGtag, {
+  config: { id: "G-30X50JL2XS" }
```

Comment:
I'm not familiar with the details of GA. Is this id considered sensitive data that outside users should not know? I.e., if someone has access to this, could they spam our GA instance with false data?

Reply:
These are public for all sites, but we can set up filtering before handoff if there's concern about someone hijacking the number to inflate visitor data. Thanks for checking. 

Filtering details: https://stackoverflow.com/questions/49419250/is-sharing-google-analytics-tracking-id-safe#:~:text=2%20Answers&text=It%20will%20be%20anyway%20publicly,you%20are%20prepared%20and%20protected.

Reply:
Cool, sounds good. Approved!